helpers/test/colour-contrast.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
```

Comment:
This is the first helper that has its own directory -- I think the tests should go inside its directory just like we've done with `components`.